tree/10_ncf_internals/list-compatible-inputs

```diff
@@ -12,7 +12,7 @@ then
 fi
 
 # add a default path since it is emptied by cfengine, and busybox for android
-PATH="/sbin:/bin:/usr/sbin:/usr/bin:/usr/locan/sbin:/usr/local/bin:/system/xbin:${PATH}"
+PATH="/usr/gnu/bin:usr/gnu/x86_64-pc-solaris2.11/bin:/sbin:/bin:/usr/sbin:/usr/bin:/usr/locan/sbin:/usr/local/bin:/system/xbin:${PATH}"
```

Comment:
missing leading / in second path?
also what is 2.11?

Reply:
what is /usr/locan/bin ? (not in the change, but in the line)

Reply:
good spot @ncharles

Reply:
erf, lets say useless